phobos/model/inertia.py

```diff
@@ -352,10 +352,11 @@ def calculateMeshInertia(mass, data):
         tri_normal = triangle.normal
         tri_centre = triangle.center
         ref_tri_vector = tri_centre - origin
-        normal_angle = ref_tri_vector.angle(tri_normal, 90)
-        if normal_angle > 90:
+        # angle returns the angle in radians
+        normal_angle = ref_tri_vector.angle(tri_normal, math.pi/2.0)
+        if normal_angle > math.pi/2.0:
             sign = -1
-        elif normal_angle == 90:
+        elif normal_angle == math.pi/2.0:
```

Comment:
Checking floats for equality is problematic, though Python 3.5 does provide a solution to it, see: [https://stackoverflow.com/questions/5595425/what-is-the-best-way-to-compare-floats-for-almost-equality-in-python](https://stackoverflow.com/questions/5595425/what-is-the-best-way-to-compare-floats-for-almost-equality-in-python)

However, at the point where this becomes problematic, `normal_angle` differs so slightly from 90° that the volume of the resulting tetrahedron is negligible and so the sign shouldn't matter. Correct?
Thus this could be simplified to:

    sign = 1 if normal_angle < math.pi/2.0 else -1

If this wild conjecture is correct, please update your pull request.